deploy-deps.sh

```diff
@@ -169,6 +171,10 @@ deploy_kyverno() {
     kubectl apply -k "${script_path}/dependencies/kyverno/policy"
 }
 
+deploy_konflux_info() {
+    kubectl apply -k "${script_path}/konflux-ci/konflux-info"
```

Comment:
I think we should avoid deploying stuff from the konflux-ci dir inside deploy-deps.sh. We should either move it to the other script or move the manifests to the dependencies dir.

Reply:
right, good catch